prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -188,6 +194,72 @@ public async Task<IActionResult> UpdateSiteCompleted(int siteId)
             return NoContent();
         }
 
+        // PUT: api/Sites/5/adjudicator
+        /// <summary>
+        /// Add a site's assigned adjudicator.
+        /// </summary>
+        /// <param name="siteId"></param>
+        /// <param name="adjudicatorId"></param>
+        [HttpPut("{siteId}/adjudicator", Name = nameof(SetSiteAdjudicator))]
+        [Authorize(Policy = AuthConstants.ADMIN_POLICY)]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
```

Comment:
These endpoints dont return BadRequest anywhere, so perhaps remove this line?

Reply:
k I'll update the enrollee endpoints as well

Reply:
Updated